docs/src/modules/java/pages/model-provider-details.adoc

```diff
@@ -19,18 +19,30 @@ See link:_attachments/api/akka/javasdk/agent/ModelProvider.Anthropic.html[`Model
 
 Configuration properties:
 
-[source,java]
+[source,hocon]
 ----
 include::example$resources/reference.conf[tag=googleai-gemini]
 ----
 
 See link:_attachments/api/akka/javasdk/agent/ModelProvider.GoogleAIGemini.html[`ModelProvider.GoogleAIGemini`] for programmatic settings.
 
+== HuggingFace
+
+Configuration properties:
+
+[source,hocon]
+----
+include::example$resources/reference.conf[tag=hugging-face]
+----
+
+See link:_attachments/api/akka/javasdk/agent/ModelProvider.HuggingFace.html[`ModelProvider.HuggingFace`] for programmatic settings.
+
+
 == LocalAI
 
 Configuration properties:
 
-[source,java]
+[source,hocon]
```

Comment:
Possibly no formatter for hocon available, but at least it doesn't screw up formatting like it did when saying the snippet is Java for me. (Ruby has historically worked but also didn't work well)

Reply:
if you search for `.conf[tag` you see that we have used json, conf, xml
I don't know which one is best, but maybe you can change them all to the same?